src/main/python/deep_qa/training/trainer.py

```diff
@@ -554,4 +610,55 @@ def _get_custom_objects(cls):
         If you've used any Layers that Keras doesn't know about, you need to specify them in this
         dictionary, so we can load them correctly.
         """
-        return {}
+        return {
+                "DeepQaModel": DeepQaModel,
```

Comment:
This is the only line you should keep; the rest should go in whatever class actually uses these layers.